app.js

```diff
@@ -1,46 +1,29 @@
-import express from 'express';
 import fetch from 'node-fetch';
 import dotenv from 'dotenv';
 import { Bot } from 'grammy';
 
 // Загружаем переменные среды из файла .env
 dotenv.config();
 
-/*
-// Создаем экземпляр приложения Express
-const app = express();
-const PORT = 3000;
-
-// Определяем маршрут для корневого URL
-app.get('/', (req, res) => {
-    res.send('Hello s8');
-});
-
-// Запускаем сервер Express на порту 3000
-app.listen(PORT, () => console.log(`My server is running on port ${PORT}`));
-*/
 // Получение токена бота из переменных среды
 const BOT_TOKEN = process.env.TCC_BOT_TOKEN;
 
 // ID вашего канала
 const CHANNEL_ID = +process.env.CHANNEL_ID;
 console.log(CHANNEL_ID);
+
+//получение текста комментария из переменной окружения
+const My_Text_JSON = process.env.COMMENT_TEXT_JSON;
+        
+//разбор JSON-строки
+const data = JSON.parse(My_Text_JSON);
+
+//получение текста для комментария согласно правилам форматирования
+const textMessage = data.comment_rules;
```

Comment:
Зачем тут JSON? Мы же можем сразу получать строку
```suggestion
const { COMMENT_TEXT } = process.env;
```

Reply:
В прошлый раз у меня, почему-то, строка выдавалась не отформатированной, когда просто приходила их .env
И Chat GPT рекомендовал такое решение. Я посмотрел на него и из пояснений GPT понял, что в .env хранится всё в формате текста HTML операторы воспринимаются как текст.
Тогда мне это показалось логичным.
Что надо разобрать строку, а потом собрать с учётом правил.
Но сейчас исправил и код получает комментарий из .env в строковом формате и через parse_mode:  'HTML' форматируется нужным образом.
Почему в прошлый раз пошло не так - не понятно.